docs/particles/decorators.rst

```diff
@@ -8,20 +8,21 @@ Decorators
 Passing `~plasmapy.particles.Particle` objects to functions and methods
 =======================================================================
 
-When calculating plasma parameters, we very frequently need to access
+When calculating plasma parameters, we frequently need to access
 the properties of the particles that make up that plasma. The
-`~plasmapy.particles.decorators.particle_input` decorator allows functions
-and methods to easily access properties of different particles.
+`~plasmapy.particles.decorators.particle_input` decorator allows
+functions and methods to easily access properties of different particles.
 
-The `~plasmapy.particles.particle_input` decorator takes valid
+The `~plasmapy.particles.decorators.particle_input` decorator takes valid
```

Comment:
I expanded a few ReST links in `particle_class.py` because I was getting an error with the docs that did not happen to say which file the error was in.  So, a little bit of scope creep here, but these should be quick to review.  (Turns out the error was because I had half of a link in my incomplete change log entry.)

Reply:
I thought I got all of these with PR #1105.  I guess not 😬. If you haven't done so yet, it would be good to merge master into this PR because #1105 did touch a lot of these files.

Reply:
Yep, I did that earlier.  And with changes like that which are hard to search for and automate fixes, it's only natural to miss some.